libkbfs/folder_block_ops.go

```diff
@@ -818,7 +826,7 @@ func (fbo *folderBlockOps) fixChildBlocksAfterRecoverableError(
 		return
 	}
 
-	topBlock, err := bcache.Get(file.tailPointer(), fbo.branch())
+	topBlock, err := fbo.getBlockFromCache(file.tailPointer(), fbo.branch())
```

Comment:
the top block should be dirty, right? If so, I think it's better to explicitly fetch from the dirty cache only.